cmd/status.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"log"
+	"time"
+
+	"github.com/allcloud-io/clisso/aws"
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	RootCmd.AddCommand(cmdStatus)
+	cmdStatus.Flags().StringVarP(
+		&writeToFile, "write-to-file", "w", "",
+		"Write credentials to this file instead of the default ($HOME/.aws/credentials)",
+	)
+	viper.BindPFlag("global.credentials-path", cmdStatus.Flags().Lookup("write-to-file"))
+}
+
+var cmdStatus = &cobra.Command{
+	Use:   "status",
+	Short: "show currently valid profiles",
```

Comment:
Please keep the capitalization consistent... It looks bad if some commands look different:

```
Usage:
  clisso [command]

Available Commands:
  apps        Manage apps
  get         Get temporary credentials for an app
  help        Help about any command
  providers   Manage providers
  status      show currently valid profiles
  version     Show version info
```